app/graphql/types/concerns/test_results.rb

```diff
@@ -6,11 +6,13 @@ module TestResults
     extend ActiveSupport::Concern
 
     def score(args = {})
-      latest_test_result_batch(args).then do |latest_test_result|
-        if latest_test_result.blank?
-          0
-        else
-          latest_test_result.score
+      ::Rails.cache.fetch("#{system_id(args)}/#{object.id}/score") do
```

Comment:
If we're using ids, why do we need this hierarchical cache path? In this case, we could just cache `"#{object.id}/score"`, right?